src/index.ts

```diff
@@ -38,6 +38,19 @@ app.post("/create-room", async (c) => {
     return c.json({ roomId });
 });
 
+// Add question, return id
+app.post("/create-question", async (c) => {
+    const { Question, PossibleAnswers, CorrectAnswer, QuestionType } = 
```

Comment:
[NIT], this formatting of these two lines `const {blah, blah} = await json`is a bit unclear on first read, a slight reformat to make it more obvious would be helpful